instabot/following_service.py

```diff
@@ -34,15 +34,15 @@ def __init__(self, client, configuration):
                 await asyncio.sleep(10)
 
     async def _follow(self):
-        '''
+        """
         @raise APIError
         @raise APIJSONError
         @raise APILimitError
-        '''
+        """
         unfollowing_threshold = datetime.datetime.utcnow() - \
             self._following_timedelta
         for user in User.select().where(
-                User.was_followed_at == None,
+                        User.was_followed_at is None,
```

Comment:
I have paid some attention to that but I did guess that you've checked that you're able to use `is` operator here but [you can't](http://docs.peewee-orm.com/en/latest/peewee/querying.html#query-operators).